### ISSUE TITLE
status of Conda integration

### ISSUE THREAD

**Issue by jsierles** [pjotrp/guix-notes#14]
Is this something you're still working on? I managed to get conda installed using https://github.com/fredmanglis/guix-conda. But curious: what's the binary rewriting in your notes is for? Would you be sharing any library paths between conda and python in guix?

**Comment by pjotrp** [comment]
@fredmanglis Conda should go into mainline Guix soon. Thanks for trying!

Binary rewrites is for installing standard Guix binaries without root permissions. It works - one thing I need to do is make sambamba available as a rewritable package so it can be installed with conda as a binary. At this moment I only use it for the D compiler on an HPC.

**Comment by fredmanglis** [comment]
@jsleries Yes. I'm currently working on getting conda on mainline Guix. I'm currently trying to get the auxlib package on Guix, then after that, the ruamel.yaml package.

@pjotrp Hmm, does that mean someone else is working on it? Should I keep at it, or just ignore it and focus on other things?

**Comment by jsierles** [comment]
Thanks for the updates. @fredmanglis for ruamel.yaml, I tried running the tests after the install phase, but no luck. Since guix python is required to install conda in the first place, will guix python be able to use conda-installed packages?

**Comment by pjotrp** [comment]
@fredmanglis you are the only one working on this.

@jsierles conda is bootstrapped on Guix using guix-python. After bootstrap conda continues on its own installing python in user land - possibly using virtualenv. 

The great thing about the bootstrap is that we get something that is identical between systems (whether running on Debian or CentOS). The primary weakness of Conda is that the bootstrap is done on shifting sands. You are lucky if it works, you are lucky if you can replicate someone else's problem. 

With Guix we should have a primary bootstrap which can easily be replicated. Everything conda does on top of that should be replicatable too (to a binary  level). That is the general idea. Guix+bioconda should give everyone a full bioinformatics platform. That is what we are aiming for here.
